index.js

```diff
@@ -0,0 +1,41 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let h1 = document.createElement("h1");
+  let form = document.createElement("form");
+  let ul = document.createElement("ul");
+  let userInput = document.createElement("input");
+  let para = document.createElement("p")
+  let submitButton = document.createElement("button");
+  
+  userInput.placeholder = "Type a todo";
```

Comment:
This looks good, consider adding comments  about what these do for later use reference